docs/configuration/index-config.md

```diff
@@ -171,45 +171,48 @@ fast: true
 | Variable      | Description   | Default value |
 | ------------- | ------------- | ------------- |
 | `description` | Optional description for the field. | `None` |
-| `stored`    | Whether value is stored in the document store | `true` |
-| `indexed`   | Whether value is indexed | `true` |
-| `fast`      | Whether value is stored in a fast field | `false` |
+| `stored`    | Whether the field values are stored in the document store | `true` |
+| `indexed`   | Whether the field values are indexed | `true` |
+| `fast`      | Whether the field values are stored in a fast field | `false` |
 
 #### `datetime` type
 
-The `datetime` type can accepts multiple formats and a storage precision. The following formats are supported but need to be explicitly requested via configuration.
-- `rfc3339`, `rfc2822`, `iso8601`: Parsing dates using standard specified formats.
-- `strftime`: Parsing dates using the Unix [strftime](https://man7.org/linux/man-pages/man3/strftime.3.html) format.
-- `unix_ts_secs`, `unix_ts_millis`, `unix_ts_micros`: Parsing dates from numbers (timestamp). Only one can be used in configuration. `unix_ts_secs` is added to the list by default if none is specified. 
+The `datetime` type handles dates and datetimes. Quickwit supports multiple input formats configured independently for each field. The following formats are natively supported:
+- `iso8601`, `rfc2822`, `rfc3339`: parse dates using standard ISO and RFC formats.
+- `strftime`: parse dates using the Unix [strftime](https://man7.org/linux/man-pages/man3/strftime.3.html) format.
+- `unix_ts_secs`, `unix_ts_millis`, `unix_ts_micros`: parse Unix timestamps. At most one Unix timestamp format must be specified.
```

Comment:
Silly question...  Should we have a `unix_ts` format without any units?

Timestamps are actually rather ambiguous.

Reply:
Yes, I will open an issue, and we can tackle that in a separate PR.